pkg/cli/config/config.go

```diff
@@ -42,6 +42,7 @@ type BackplaneConfiguration struct {
 	JiraConfigForAccessRequests AccessRequestsJiraConfiguration `json:"jira-config-for-access-requests"`
 	VPNCheckEndpoint            string                          `json:"vpn-check-endpoint"`
 	ProxyCheckEndpoint          string                          `json:"proxy-check-endpoint"`
+	DisplayClusterInfo          bool                            `json:"display-cluster-info"`
```

Comment:
As all the setting flags in the backplane configuration are currently defined as string values, but `DisplayClusterInfo` is defined as a bool, it would be better to emphasize that end-user should use the boolean value true instead of the string "true" in the `README.md`.

**P.S** This might not be necessary, if you'd like to go a step further, you could implement a small utility function to handle both boolean and string values for `display-cluster-info`, basically, if the value is a string (`"true" or "false"`), it is converted and interpreted as a boolean.